js/card-options.js

```diff
@@ -0,0 +1,25 @@
+var CarLot = (function (loadCars) {
+  var userInput = document.getElementById('user-input');
+
+  //Changes card border thickness and background color
+  loadCars.changeCardAttr = function (clickEvent) {
+  clickEvent.classList.add("card-change"); //add class
+  };
+
+  //Blanks the User Input and then sets focus on that input box
+  loadCars.setFocus = function () {
+    userInput.value = "";
+    userInput.focus();
+  };
+
+  //changes back to initial load border thickness and bgcolor
+  loadCars.resetCard = function (clickEvent) {
+    var cards = document.querySelectorAll('.card-style');
+      for (i = 0; i < cards.length; i++) {
+        cards[i].classList.remove("card-change");
+      };
```

Comment:
This for loop should be unindented a step, to the same level as the ```var``` declaration on line 17